src/Request/AbstractRequest.php

```diff
@@ -77,6 +98,11 @@ public function getParameters()
             $parameters['NUMAPPEL'] = $this->getCallNumber();
         }
 
+        // Direct Plus requests special case.
+        if (null !== $this->getSubscriberRef()) {
+            $parameters['REFABONNE'] = $this->getSubscriberRef();
```

Comment:
Well, `$parameters` are going to be directly send to the request. How can I translate it? Oo